package.json

```diff
@@ -14,10 +14,11 @@
   "dependencies": {
     "@babel/compat-data": "^7.8.6",
     "@bcrs-shared-components/certify": "^1.0.3",
+    "@bcrs-shared-components/contact-info": "1.0.3",
```

Comment:
Please bump up the app patch version.

Reply:
1.0.3 is the latest! or do you mean something else?

Reply:
line 3

Reply:
ah, sorry. I misunderstood, thought you were referring to this import.